cypress/helpers/select-typeahead-input.ts

```diff
@@ -0,0 +1,7 @@
+export function selectTypeaheadInput(testId: string, value: string) {
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(1000); // TODO: we shouldn't need this wait, but the app fails to type the document type without it
```

Comment:
Just curious, does this reliably pass with a wait time of less than a second?

Reply:
🤔 I'll investigate and update you.